docs/configuring-output.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[output-config]]
+== Output Configuration
+
+An output direct data from APM Server to the desired storage system.
+A typical deployment uses Elasticsearch as the output.
+Options like Logstash and Kafka are popular with users requiring additional data enrichment and highly scalable setups.
+
+[float]
+=== Sourcemaps
+
+Sourcemaps can be uploaded through various outputs but can currently only be applied when Elasticsearch is the output.
```

Comment:
If you upload a sourcemap to kafka, and then change the output to elasticsearch, the sourcemap can't be applied, right?
So i'd say they can only be uploaded and applied to Elasticsearch.

Reply:
If you upload via kafka and then change the output to elasticsarch the sourcemap will be applied.  If you leave the output as kafka, the sourcemap can not be retrieved.  Thanks for opening #1104 to discuss this.